Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
```

Comment:
Having this `Gemfile` at the top-level of the vagrant directory seems to clutter the project. Is it possible to move this file into `spec`?